partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/FnBitwiseAnd.kt

```diff
@@ -8,9 +8,10 @@ import org.partiql.spi.value.Datum
 import org.partiql.types.PType
 import kotlin.experimental.and
 
-internal object FnBitwiseAnd : ArithmeticDiadicOperator() {
-    override fun getName(): String {
-        return "bitwise_and"
+internal object FnBitwiseAnd : DiadicArithmeticOperator("bitwise_and") {
+
+    init {
+        fillTable()
     }
```

Comment:
nit and don't change anything, but I think the pattern here is "must call super" which is fairly common and might have some annotations to compile-time enforce this, but I wouldn't change it